go/tests/endpoint/endpoint.go

```diff
@@ -54,6 +54,20 @@ type fn func()
 
 var cleanup fn = func() {}
 
+var cwd, err_cwd = os.Getwd()
+var bin, err_bin = os.Executable()
+
+func init() {
```

Comment:
i think we should alphabetize init() since the rest of endpoint's member funcs are arranged alphabetically now.